dex/src/matching.rs

```diff
@@ -1092,6 +1127,35 @@ impl<'ob> OrderBookState<'ob> {
         Ok(())
     }
 
+    pub(crate) fn boot_order(
```

Comment:
can you use this function where they boot orders in new order?

Reply:
you could but there are other issues with borrowing self multiple times because within the scope of this loop self is already used to get self.orders_mut(side)